src/View/index.js

```diff
@@ -0,0 +1,162 @@
+      borderTop: OIOResponsiveObjectPropType,
+      borderRadius: OIOResponsiveObjectPropType,
+      borderRight: OIOResponsiveObjectPropType,
+      borderBottom: OIOResponsiveObjectPropType,
+      borderLeft: OIOResponsiveObjectPropType,
+      boxShadow: OIOResponsiveObjectPropType,
+      children: PropTypes.node,
+      className: PropTypes.string,
+      display: OIOResponsiveObjectPropType,
+      flex: OIOResponsiveObjectPropType,
+      flexFlow: OIOResponsiveObjectPropType,
+      float: OIOResponsiveObjectPropType,
+      height: OIOResponsiveObjectPropType,
+      justifyContent: OIOResponsiveObjectPropType,
+      margin: OIOResponsiveObjectPropType,
+      marginTop: OIOResponsiveObjectPropType,
+      marginLeft: OIOResponsiveObjectPropType,
+      marginRight: OIOResponsiveObjectPropType,
+      marginBottom: OIOResponsiveObjectPropType,
+      minHeight: OIOResponsiveObjectPropType,
+      maxHeight: OIOResponsiveObjectPropType,
+      minWidth: OIOResponsiveObjectPropType,
+      maxWidth: OIOResponsiveObjectPropType,
+      onClick: PropTypes.func,
+      onKeyUp: PropTypes.func,
+      onScroll: PropTypes.func,
+      order: OIOResponsiveObjectPropType,
+      overflow: OIOResponsiveObjectPropType,
+      padding: OIOResponsiveObjectPropType,
+      paddingTop: OIOResponsiveObjectPropType,
+      paddingLeft: OIOResponsiveObjectPropType,
+      paddingRight: OIOResponsiveObjectPropType,
+      paddingBottom: OIOResponsiveObjectPropType,
+      paddingHorizontal: OIOResponsiveObjectPropType,
+      paddingVertical: OIOResponsiveObjectPropType,
+      position: OIOResponsiveObjectPropType,
+      scroll: OIOResponsiveObjectPropType,
+      style: PropTypes.object,
+      textAlign: OIOResponsiveObjectPropType,
+      top: OIOResponsiveObjectPropType,
+      left: OIOResponsiveObjectPropType,
+      right: OIOResponsiveObjectPropType,
+      bottom: OIOResponsiveObjectPropType,
+      WebkitOverflowScrolling: OIOResponsiveObjectPropType,
+      width: OIOResponsiveObjectPropType,
+      zIndex: OIOResponsiveObjectPropType
+   }
+
+   static defaultProps = {
+      className: '',
+      display: r`block`,
+      onClick: () => {},
+      onKeyUp: () => {},
+      onScroll: () => {},
+      position: r`relative`,
+      style: {}
+   }
+
+   render() {
+      const {
+         children, className,
+         display, float, position, top, left, right, bottom,
+         flex, flexFlow, justifyContent, alignItems, order,
+         height, width, maxHeight, maxWidth, minHeight, minWidth,
+         margin, marginBottom, marginLeft, marginRight, marginTop,
+         padding, paddingBottom, paddingLeft, paddingRight, paddingTop,
+         border, borderBottom, borderLeft, borderRight, borderTop,
+         backgroundColor, backgroundImage, backgroundPosition, backgroundSize,
+         borderRadius, boxShadow, textAlign, zIndex,
+         overflow, WebkitOverflowScrolling,
+         onClick, onKeyUp, onScroll
+      } = this.props
```

Comment:
What if you want to pass other properties to the `div`, such as the ones mentioned in the spec: https://www.w3.org/TR/html401/struct/global.html#h-7.5.4

Reply:
As discussed, add `id`. And maybe some of the important events?

Reply:
Done.